docs/content/preview/yugabyte-platform/configure-yugabyte-platform/supported-os-and-arch.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Supported operating systems and architectures
+headerTitle: Supported operating systems and architectures
+linkTitle: OS support
+description: List of operating systems and architectures supported by YugabyteDB Anywhere
+menu:
+  preview_yugabyte-platform:
+    identifier: supported-os-and-arch
+    parent: configure-yugabyte-platform
+    weight: 6
+type: docs
+---
+
+## Supported CPU architectures
+
+YugabyteDB Anywhere supports deploying YugabyteDB on both x86 and ARM (aarch64) architecture-based hardware.
+
+## Supported operating systems
+
+YugabyteDB Anywhere supports deploying YugabyteDB on the following operating systems:
+
+* AlmaLinux OS 8
+* CentOS 7
+* Oracle Linux 8
+* Ubuntu 18
+* Ubuntu 20
+
+### Requirements for all OSes
+
+Python 3 is required. If you're using YugabyteDB Anywhere to provision nodes in public clouds, be sure the custom AMI you plan to use has Python 3 installed.
+
+Either `gtar` or `gunzip` must be present on the host AMI.
```

Comment:
Did some more digging into the Ansible [module](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/unarchive_module.html#notes) and we need all of the following: 
zipinfo
gtar

Unzip is not required as we don't use/distribute any zip files.

Reply:
Updated!